docs/proposal/observability.md

```diff
@@ -0,0 +1,274 @@
+\"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\" %UPSTREAM_CLUSTER% %UPSTREAM_LOCAL_ADDRESS% %DOWNSTREAM_LOCAL_ADDRESS% %DOWNSTREAM_REMOTE_ADDRESS% %REQUESTED_SERVER_NAME% %ROUTE_NAME%\n
+```
+
+- %START_TIME%: Request start time, used to calculate request duration.
+- %REQ(:METHOD)%：Request method, such as GET, POST etc.
+- %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)%：The URL path of the request, excluding the host and port information.
+- %PROTOCOL%：The network protocol and version used for the request.
+- %RESPONSE_CODE%：The HTTP status code returned by the server when responding to the request, such as 200, 404 etc.
+- %RESPONSE_FLAGS%：Response flags indicating additional information like whether the response included a content-length or was a chunked response.
+- %RESPONSE_CODE_DETAILS%: More detailed explanation of the status code, such as the specific reason for a 4xx error.
+- %CONNECTION_TERMINATION_DETAILS%: How the connection was terminated, such as idle timeout or proxy timeout.
+- %UPSTREAM_TRANSPORT_FAILURE_REASON%:
+- %BYTES_RECEIVED%: The total number of bytes received by the proxy from the downstream client.
+- %BYTES_SENT%: The total number of bytes sent by the proxy to the upstream host.
+- %DURATION%: The total time taken for the request to pass through the proxy from receipt to response, in milliseconds.
+- %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)%: The actual time taken by the upstream service to process the request and generate the response.
+- %REQ(X-FORWARDED-FOR)%: The original client IP address, with each proxied hop's IP appended.
+- %REQ(USER-AGENT)%: The client's user-agent software and version identification string.
+- %REQ(X-REQUEST-ID)%: A unique identifier for the request.
+- %REQ(:AUTHORITY)%: The value of the HTTP request's Host header field.
+- %UPSTREAM_HOST%: The real IP address or hostname of the upstream host.
+- %UPSTREAM_CLUSTER%: The name of the cluster that the upstream host belongs to.
+- %UPSTREAM_LOCAL_ADDRESS%: The local IP address of the proxy for its connection to the upstream node.
+- %DOWNSTREAM_LOCAL_ADDRESS%: The IP and port on which the proxy listens for client connections.
+- %DOWNSTREAM_REMOTE_ADDRESS%: The IP and port on which the proxy listens for client connections.
+- %REQUESTED_SERVER_NAME%: The server name specified in the request.
+- %ROUTE_NAME%: The name of the route that the request matched and used.
+
+These metrics are all provided by envoy.
+
+In practice, isito gets the accesslog as follows:
+
+```console
+sleep: "GET /status/418 HTTP/1.1" 418 - via_upstream - "-" 0 135 10 10 "-" "curl/8.7.1" "26bb8053-cfc8-4aa1-aa26-ae48f8ef1281" "httpbin:8000" "envoy://connect_originate/10.244.0.19:80" outbound|8000||httpbin.default.svc.cluster.local envoy://internal_client_address/ 10.96.62.92:8000 10.244.0.18:53560 - default
```

Comment:
If you want to focus on L4 accesslog, you should paste what istio L4 log look like